torch/csrc/jit/codegen/cuda/scheduler.h

```diff
@@ -13,20 +14,68 @@ TORCH_CUDA_API bool scheduleFusion(
     Fusion* fusion,
     const at::ArrayRef<c10::IValue> inputs);
 
+/**
+ * This struct communicates whether a launch param
+ * should be calculated per kernel instance if a new set of elements
+ * matches the kernel.
+ *
+ * @param value_ : -1 = symbolic, > 0 = valid constant launch param
+ * @param symbolic_value_ : a pointer to a symbolic val of param
+ */
+struct LaunchParam {
+  bool mutable_ = false;
```

Comment:
_ suffix is only for private data members. (since mutable is a reserved keyword, it will have to be renamed to something like `is_mutable` ...)